src/page-modules/assistant/trip/trip-pattern/index.tsx

```diff
@@ -87,7 +88,8 @@ export default function TripPattern({
   return (
     <div className={style.tripPatternContainer}>
       <motion.div
-        role={'button'}
+        id="assistant-details-accordion"
+        role={'region'}
```

Comment:
```suggestion
        role="region"
```

Reply:
Possibly also this section should have a `aria-labelledby` with reference to the action item

Reply:
Hmm. Maybe not, as the header it self is visible further down